### PR TITLE
folder naming etc in examples is corrected

### LINKED ISSUE #256: Consistent data_folder naming in examples

### PR DESCRIPTION
**Pull request by agsedova**
close #256

### CODE REVIEW

File: examples/utils.py [hunk 1]
```diff
@@ -38,13 +40,55 @@ def get_samples_list(data: Union[pd.Series, pd.DataFrame], column_num: int = Non
...
+
+
+def convert_text_to_transformer_input(tokenizer, texts: List[str]) -> TensorDataset:
+    """
+    Convert input data to BERT encoded features (more details could be found at
+    https://huggingface.co/transformers/model_doc)
+    :param texts: training/dev/test samples that are to be encoded with BERT features. Can be given as Series or
+    as DataFrames with specified column number where the sample are stored.
+    :param tokenizer: DistilBertTokenizer tokenizer for english from HuggingFace
+    :param column_num: optional parameter that is needed to specify in which column of input_data Dataframe the samples are stored
```
Comment (marina-sp) [inline examples/utils.py:85]: Also not defined and not used

File: examples/utils.py [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import os
-from typing import Union, Tuple, List
+from numbers import Integral
```
Comment (marina-sp) [inline examples/utils.py:2]: Do we need this?

File: examples/trainer/wscrossweigh/wscrossweigh_training_tutorial.py [hunk 3]
```diff
@@ -28,38 +29,58 @@ def train_wscrossweigh(path_to_data: str, num_classes: int) -> None:
...
-        dev_dataset_bert = get_bert_encoded_features(dev_df["sample"].tolist(), tokenizer)
-        dev_labels = TensorDataset(LongTensor(list(dev_df.iloc[:, 1])))
-    else:
-        dev_labels, dev_dataset_bert = None, None
+    X_train = convert_text_to_transformer_input(df_train["sample"].tolist(), tokenizer)
+    X_dev = convert_text_to_transformer_input(df_dev["sample"].tolist(), tokenizer)
+    X_test = convert_text_to_transformer_input(df_test["sample"].tolist(), tokenizer)
+
+    y_dev = TensorDataset(LongTensor(list(df_dev.iloc[:, 1])))
+    y_test = TensorDataset(LongTensor(list(df_test.iloc[:, 1])))
```
Comment (marina-sp) [inline examples/trainer/wscrossweigh/wscrossweigh_training_tutorial.py:75]:
```suggestion
    y_dev = TensorDataset(LongTensor(df_dev.iloc[:, 1].tolist()))
    y_test = TensorDataset(LongTensor(df_test.iloc[:, 1].tolist()))
```

But also why not access the column by name with df["label"]?

**Review by marina-sp — CHANGES_REQUESTED** [review]

File: examples/trainer/wscrossweigh/wscrossweigh_training_tutorial.py (hunk 3 above)
Comment (agsedova) [reply examples/trainer/wscrossweigh/wscrossweigh_training_tutorial.py:75]: Ah apparantly somehow wasn't corrected last time we discussed it. done.

**Comment by marina-sp** [comment]
@agsedova btw i did not look at the jupyter notebook, is the anything specific to look at or does it just mirror the python script?

**Pull request merged**